src/_sass/_main_nav.scss

```diff
@@ -0,0 +1,532 @@
+          width: 0;
+        }
+
+        ul:after {
+          border: 0.5rem solid transparent;
+          border-bottom-color: $white;
+          left: calc(50% - 0.5rem);
+          top: 0.625rem;
+        }
+
+        ul:before {
+          border: 0.5625rem solid transparent;
+          border-bottom-color: $mobile-nav-border;
+          left: calc(50% - 0.5625rem);
+          top: 0.5rem;
+        }
+
+        p {
+          margin: 0;
+          padding: 0;
+        }
+
+        li:hover {
+          background-color: $hover-color;
+          .nav-link-text {
+            color: $nav-blue;
+          }
+        }
+      }
+      
+      .underlineable {
+        position: relative;
+        color: #000;
+        text-decoration: none;
+      }
+
+      .underlineable:before {
+        content: "";
+        position: absolute;
+        width: 100%;
+        height: 2px;
+        bottom: 0;
+        left: 0;
+        background-color: $nav-blue;
+        visibility: hidden;
+        -webkit-transform: scaleX(0);
+        transform: scaleX(0);
+        -webkit-transition: all 0.3s ease-in-out 0s;
+        transition: all 0.3s ease-in-out 0s;
+      }
+      
+      // .reveal:after {
+      //   z-index: 0;
+      //   content: "";
+      //   position: relative;
+      //   margin-top: 50px;
+      //   height: 1px;
+      //   width: 0px;
+      //   background-color: red;
+      //   transition: width .2s ease;
+      // }
+
+      .reveal:hover {
+        .bottom-border {
+          width: 100%;
+        }
+
+        .underlineable:before {
+          visibility: visible;
+          -webkit-transform: scaleX(1);
+          transform: scaleX(1);
+        }
+        // border-bottom: 0.125rem solid $nav-blue;
+        
+        .dropdown-wrapper { display: block};
+      }
+
+      .reveal:hover + .down {
+        transform: rotate(180deg);
+      }
+
+      .reveal:hover + .reveal:after {
+        width: 100%;
+      }
+      
+      .reveal, .nav-btn, .reveal *:hover {
+        cursor: pointer;
+      }
+
+      .nav-btn {
+        background-color: $nav-blue;
+        border-radius: 0.0625rem;
+        border: 0.0625rem solid $nav-blue;
+        box-sizing: border-box;
+        color: $white;
+        height: 3rem;
+        padding: 0.8125rem 1.5rem;
+      }
+      
+      
+      .nav-btn:hover {
+        background-color: $white;
+        border: 0.0625rem solid $nav-blue;
+        box-sizing: border-box;
+        color: $nav-blue;
+      }
+
+      .nav-btn-white {
+        color: $nav-blue;
+        background-color: $white;        
+      }
+
+      .nav-btn-white:hover {
+        color: $white;        
+        background-color: $nav-blue;
+      }
+
+      .login-btns {
+        display: flex;
+      }
+    }
+  }
+}
+
+@media screen and (max-height: 50rem) {
+  .mobile-nav {
+    height: 100vh;
+    overflow: auto;
+  }
+}
+
+@media screen and (max-width: $tablet-landscape) {
+  .nav-content {
+    width: 100% !important;
```

Comment:
is the !important necessary? is you use a parent class, you might be able to get to the style to override without needing this